src/Symfony/Component/Form/Extension/Validator/Type/UploadValidatorExtension.php

```diff
@@ -55,4 +57,9 @@ public function getExtendedType()
     {
         return 'form';
     }
+
+    public function __toString()
```

Comment:
-1 for making the TypeExtension itself castable to string as the error message. It looks really weird to me.